docs/user_interface.rst

```diff
@@ -25,284 +24,412 @@ If you do processing or acquisition which produces new data, its associated disp
 
-The adjustment can be set to None, Equalized, Gamma, or Log. Adjustments are applied after brightness and contrast. Equalized means the display will attempt to have more color variation where there are is intensity density. Gamma means the display will apply a gamma curve to the contrast transfer function. The values for gamma typically range from 10 to 1/10 with the default value of 1. You can enter numbers as fractions, such as "1/2". Log means the display will apply a log to the contrast transfer function. If intensity values are small or negative, the behavior is undefined.
+.. image:: graphics/inspector_image.png
+    :width: 321
+    :alt: Inspector Panel with Image Selected
+    :align: center
 
-If your data is complex, you can also choose how to convert the data from complex to a scalar value for display. The options are Log Absolute, Absolute, Real, and Imaginary. The default is Log Absolute.
+When different kinds of items are selected (display item, graphic, etc.) the Inspector's subsections will change to display settings relevant to that type of item. If an image or line plot is selected in a display panel, the inspector will provide all subsections relevant to the selected item. In the image above, an image is selected; and in the image below, a line plot is selected.
 
-.. _Layers Inspector Section:
+.. image:: graphics/inspector_line_plot.png
+    :width: 321
+    :alt: Inspector Panel with Line Plot Selected
+    :align: center
 
-Line Plot Display Layers
-++++++++++++++++++++++++
-.. layer name, move layer forward/back, add/remove layer
-.. data index, row
-.. fill color, stroke color, stroke width
-.. complex display type
+.. _Info Inspector Section:
+
+Info
+----
+.. image:: graphics/inspector_info_subsection.png
+    :width: 321
+    :alt: Inspector Info Subsection
+    :align: center
+
+The Info subsection of the inspector allows you to edit the title and description of the selected item. The Info subsection will only be visible if a single item is selected. 
+
+If the item selected contains multiple data items, like a layered line plot for example, changing the title and description of the item will not affect the names and descriptions of each data item; it will change the title and description for the combined display item.
+
+.. _Image Display Inspector Section:
+
+Image Display
+-------------
+.. image:: graphics/inspector_image_display_subsection.png
+    :width: 321
+    :alt: Inspector Image Display Subsection
+    :align: center
+
+Image Display allows you to force an image to be displayed as a line plot or as script instead of the default.
+
+.. _Image Data Inspector Section:
+
+Image Data
+----------
+.. image:: graphics/inspector_image_data_subsection.png
+    :width: 321
+    :alt: Inspector Image Data Subsection
+    :align: center
+
+Image Data presents several figures and settings for a selected image:
+
+* Date - Displays the date and time a selected image was created.
```

Comment:
I think all words referring to GUI elements, should be in :guilabel: directives. e.g. :guilabel:`Date` (note the "Date" must be enclosed in _backticks_). This goes for everything in this document.